deep_qa/data/tokenizers/character_tokenizer.py

```diff
@@ -29,8 +29,8 @@ def index_text(self,
 
     @overrides
     def embed_input(self,
-                    input_layer: 'keras.layers.Layer',
```

Comment:
so this works if we do an
`from keras.layers import Layer` and then just set the type annotation to `input_layer: Layer`. There's no inherent reason to use a forward reference here, since you can just directly import the module and then use that object as the type hint. Forward references are mainly useful when you have circular references, as is the case of `'TextDataset'`. I haven't figured out how to get sphinx to play nice with that yet.